cloud/src/meta-service/meta_service.cpp

```diff
@@ -618,11 +649,13 @@ void MetaServiceImpl::create_tablets(::google::protobuf::RpcController* controll
         msg = fmt::format("failed to get vault id, vault name={}", name);
         return;
     }
+    DCHECK(request->has_db_id());
```

Comment:
What happens if the doris FE is the old version?

Reply:
> What happens if the doris FE is the old version?

resolved